### PR TITLE
Fix product/company lazy loading

### PR DESCRIPTION
**Pull request by fgourichon**
Late PR... @ouranos FYI

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "angular-schema-form-bootstrap": "^0.2.0",
     "angular-filter": "^0.5.16",
     "ng-file-upload": "^12.2.13",
-    "angucomplete-alt": "^1.3.1",
+    "angucomplete-alt": "git://github.com/fgourichon/angucomplete-alt.git#1.3.2" ,
```
Comment (alexnoox) [inline bower.json]: @ouranos Should we move this fork of `angucomplete-alt` on a maestrano repo?

Comment (ouranos) [reply bower.json]: Yup

File: src/app/views/product-markups/create-markup-modal/create-markup.html [hunk 2]
```diff
@@ -3,32 +3,32 @@
     <h3 class="modal-title" translate>mnoe_admin_panel.dashboard.product_markups.add_markup.modal.title</h3>
   </div>
 
-  <form name="vm.form.markup" novalidate>
+  <form name="vm.form.markup">
     <div class="modal-body">
       <div class="row">
-        <angucomplete-alt id="product-selector"
+        <angucomplete-alt id="productSelector"
```
[the remark this answers is not in the document]
Comment (fgourichon) [reply src/app/views/product-markups/create-markup-modal/create-markup.html]: Yes, why?

File: bower.json (hunk 1 above)
Comment (fgourichon) [reply bower.json]: Pointed to maestrano/angucomplete-alt, I'll let you confirm when the report is available, I don't have the rights to create it.

**Pull request merged**